client/state/data-layer/wpcom/nps-survey/index.js

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	NPS_SURVEY_CHECK_ELIGIBILITY,
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { setNpsSurveyEligibility } from 'state/nps-survey/actions';
+
+export const fetchSurveyEligibility = action => ( dispatch, getState ) => {
```

Comment:
Since we're using @jsnajdr's`dispatchRequestEx` this function can return a plain action eg

```jsx
const fetchSurveyEligibility = action => http(
		{
			apiVersion: '1.2',
			method: 'GET',
			path: '/nps',
		},
		action
	);
    
```

See also p4TIVU-8LU-p2